### PR TITLE
[DOCS] Add template for code snippet under test for DQ schema use case

### PR DESCRIPTION
**Pull request by rachhouse**
- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.md [hunk 1]
```diff
@@ -228,6 +228,11 @@ print("Validation Result 1:", result_1)
 print("Validation Result 2:", result_2)
 ```
 
+Snippet:
+
+```python title="Python" name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py full sample code"
```
Comment (rachhouse) [inline docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.md:233]: In your markdown file, snippets from the `.py` script can be imported using a `python` code block and the full name of the snippet - the filepath and short description (see the `schema.py` script file & my associated comment for more details on defining the snippet name).

File: docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+"""
```
Comment (rachhouse) [inline docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py:1]: This script will get run during CI. Scripts that contain code snippets referenced in their associated docs pages can be co-located in the same directory.

File: docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+# Add test data to database for testing.
+for test_data_set in TEST_DATA_SETS:
+    load_data_into_test_database(
+        table_name=test_data_set["table_name"],
+        csv_path=str(GX_ROOT_DIR / test_data_set["test_data_path"]),
+        connection_string=CONNECTION_STRING,
+    )
+
+# <snippet name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py full sample code">
```
Comment (rachhouse) [inline docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py:41]: This comment format defines a code snippet. Any code within the snippet `<snippet></snippet>` block will be shown when you call it on the `.md` page.
* Snippet name format is the full filepath name of the script relative to the GX repo root directory (`docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py`) and a short description of the snippet (`full sample code`). Enclose the name in double quotes.

```
# <snippet name="full/file_path.py short description of snippet">
Code you want
to be shown
in the docs
# </snippet>
```

File: docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+batch_definition_2 = data_asset_1.add_batch_definition_whole_table("batch definition 2")
+batch_2 = batch_definition_2.get_batch()
+
+# Validate Batches using the Expectation Suite.
+results_1 = batch_1.validate(suite)
+results_2 = batch_2.validate(suite)
+
+print(f"Validation results 1:\n{results_1}")
+print(f"Validation results 2:\n{results_2}")
+# </snippet>
```
Comment (rachhouse) [inline docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py:82]: Note the ending block of the snippet.

File: tests/test_sets/learn_data_quality_use_cases/schema_financial_transfers_1.csv [hunk 5]
```diff
@@ -0,0 +1,3 @@
+type,sender_account_number,recipient_fullname,transfer_amount,transfer_date
```
Comment (rachhouse) [inline tests/test_sets/learn_data_quality_use_cases/schema_financial_transfers_1.csv:1]: I've created a directory for any test/sample data you create for the data quality use cases. Please add it to:
```
tests/test_sets/learn_data_quality_use_cases/
```

File: docs/docusaurus/docs/components/examples_under_test.py [hunk 6]
```diff
@@ -48,5 +51,16 @@
     # ),
 ]
 
+learn_data_quality_use_cases = [
+    # Schema.
+    IntegrationTestFixture(
```
Comment (rachhouse) [inline docs/docusaurus/docs/components/examples_under_test.py:56]: This `IntegrationTestFixture` defines a test using `docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py`.

**Review by rachhouse — COMMENTED** [review]
@Quantisan I've added some comments to help explain the template I've added here that creates a simple script used for code snippets in the docs and CI testing. Let me know if you have any questions. When you're ready, I'll merge this into your branch.

**Review by Quantisan — APPROVED** [review]

**Pull request merged**